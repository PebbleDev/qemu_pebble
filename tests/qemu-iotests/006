#!/bin/bash
#
# Make sure qemu-img rejects > 127GB images for the vpc format as the format
# doesn't support this.
#
# Copyright (C) 2009 Red Hat, Inc.
#
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.
#

# creator
owner=hch@lst.de

seq=`basename $0`
echo "QA output created by $seq"

here=`pwd`
tmp=/tmp/$$
status=1	# failure is the default!

_cleanup()
{
	_cleanup_test_img
}
trap "_cleanup; exit \$status" 0 1 2 3 15

# get standard environment, filters and checks
. ./common.rc
. ./common.filter

_supported_fmt vpc
_supported_proto generic
_supported_os Linux


echo
echo "creating 128GB image"
_make_test_img 128G

# success, all done
echo "*** done"
rm -f $seq.full
status=0
