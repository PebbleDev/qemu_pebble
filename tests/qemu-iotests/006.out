QA output created by 006

creating 128GB image
qemu-img: The image size is too large for file format 'vpc'
Formatting 'TEST_DIR/t.IMGFMT', fmt=IMGFMT size=137438953472 
*** done
